<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="CargaT">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelAdminNuevo_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Carga:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAdminNuevo_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>L. inicio :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LugarIT">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelAdminNuevo_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>H. inicio:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="HoraIT">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelAdminNuevo_7">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>190</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>H. final</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAdminNuevo_8">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Peso:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="HoraFT">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>180</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="PesoT">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="LugarFT">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>100</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelAdminNuevo_9">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>110</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>L. final:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BotonLlamada">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>360</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(226, 231, 255);</string>
    </property>
    <property name="text">
     <string>Llamar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../backup_usb-2022/documentos/Kinect/FinalVersion/buscar.svg</normaloff>../backup_usb-2022/documentos/Kinect/FinalVersion/buscar.svg</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="LabelLlamada">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>370</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Linea libre</string>
    </property>
   </widget>
   <widget class="QComboBox" name="TCamion">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="labelAdminNuevo_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tipo:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Text_pers">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>671</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="B_personalizado">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>230</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(226, 231, 255);</string>
    </property>
    <property name="text">
     <string>Personalizado</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../backup_usb-2022/documentos/Kinect/FinalVersion/buscar.svg</normaloff>../backup_usb-2022/documentos/Kinect/FinalVersion/buscar.svg</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="B_estandar">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>230</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(226, 231, 255);</string>
    </property>
    <property name="text">
     <string>Estandar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../backup_usb-2022/documentos/Kinect/FinalVersion/buscar.svg</normaloff>../backup_usb-2022/documentos/Kinect/FinalVersion/buscar.svg</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
